sdks/js/packages/core/react/components/organization/billing/invoices/index.tsx

```diff
@@ -140,7 +141,10 @@ export default function Invoices({
 
   const showLoader = isLoading || isInvoicesLoading;
 
-  const columns = getColumns({ isLoading: showLoader });
+  const columns = getColumns({
+    isLoading: showLoader,
+    dateFormat: config?.dateFormat || ''
```

Comment:
can we set default date formate here instead of ""

Reply:
sure